utils/http-utils/src/main/scala/pl/touk/nussknacker/http/backend/HttpClientConfig.scala

```diff
@@ -75,18 +82,49 @@ object HttpClientConfig {
       useNative = forOption[Boolean]("useNative"),
       followRedirect = forOption[Boolean]("followRedirect"),
       forceShutdown = forOption[Boolean]("forceShutdown"),
-      configForProcess = forOption("configForProcess")(mapValueReader(vr))
+      configForProcess = forOption("configForProcess")(mapValueReader(vr)),
+      forbiddenHosts = forOption[List[String]]("forbiddenHosts"),
     )
   })
 
 }
 
 object DefaultHttpClientConfig {
 
-  def apply(): HttpClientConfig = HttpClientConfig(None, None, None, None, None, None, None)
+  def apply(): HttpClientConfig = HttpClientConfig(None, None, None, None, None, None, None, None)
 
   val maxPoolSize: Int = 20
 
   val timeout: FiniteDuration = 10 seconds
 
+  implicit class ClientConfigBuilderExtension(val builder: DefaultAsyncHttpClientConfig.Builder) extends AnyVal {
+
+    def setForbiddenHostRequestFilter(forbiddenHosts: List[String]): DefaultAsyncHttpClientConfig.Builder =
+      if (forbiddenHosts.nonEmpty) {
```

Comment:
Why not do this Option[List[String]] as forbiddenHosts and just do map().getOrElse()? It's more TypeSafe. I think we should pass in HttpClientConfig forbiddeHost as  Option[NonEmptyList[String]], it's more representative.

Reply:
Done